london_cafes/london_cafes/templates/base.html

```diff
@@ -0,0 +1,40 @@
+{% load static %}
+
+<!DOCTYPE html>
+<html lang="en-gb">
+    <head>
+        <title>{% block title %} {% endblock %}| London Cafes</title>
+        <!--Meta Data-->
+        <meta charset="UTF-8">
+        <meta name=viewport content="width=device-width, initial-scale=1">
+        <meta name="created" content='2nd July 2016 04:37' />
+        <meta name="description" content="London Cafes" />
+        <meta name="author" content="London Cafes in Partnership with Women Hack for Non-Profits" />
+        <meta name="copyright" content="All text content and images copyright London Cafes" />
+        <meta name="company" content="London Cafes" />
+
+        <!-- Latest compiled and minified CSS -->
+        <link href="{% static "styles/styles.css" %}" rel="stylesheet">
+        <!-- <link rel="shortcut icon" href="{% static "img/favicon.png" %}" type="image/x-icon"> -->
+    </head>
+    <body>
+      <div class="body-wrap">
+
+          {% block layout %} {% endblock %}
+
+      </div>
+
+      <!-- Latest compiled and minified JavaScript -->
+
+      <!--[if IE]><script type="text/javascript" src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
+
+      <script>
+         (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+             (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+             m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+         })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
+         ga('create', 'UA-76152092-1', 'auto');
```

Comment:
I didn't I meant to comment about it and I forgot. I will remove the GA number :)